### PR TITLE
setserial: add page

### LINKED ISSUE #8131: page request: setserial
### Command description

Let's document [setserial](https://linux.die.net/man/8/setserial)

### Command details

_No response_

### OS

Linux

### OS details

_No response_

### PR DESCRIPTION
**Pull request by DEBADRIBASAK**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
Closes #8131

### CODE REVIEW

File: pages/linux/setserial.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# setserial
+
+> Read or modify a LInux serial port information.
+> More information: <https://linux.die.net/man/8/setserial>.
```
Comment (navarroaxel) [inline pages/linux/setserial.md:4]: We prefer manned
```suggestion
> More information: <https://manned.org/setserial>.
```

File: pages/linux/setserial.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# setserial
+
+> Read or modify a LInux serial port information.
```
Comment (navarroaxel) [inline pages/linux/setserial.md:3]:
```suggestion
> Read and modify serial ports information.
```

File: pages/linux/setserial.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`setserial -b {{device}}`
+
+- Set configuration parameters to a device by passing commandline arguments. Superuser privilege is required in most cases:
+
+`sudo setserial {{device}} {{parameter}}`
+
+- Read configuration of a list of devices by using the `-g` option:
+
+`setserial -g {{device1}} {{device2}} ...`
```
Comment (navarroaxel) [inline pages/linux/setserial.md:20]: We use this format
```suggestion
`setserial -g {{device1 device2 ...}}`
```

File: pages/linux/setserial.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Print a summary of confguration of the serial device (useful for printing during bootup process):
+
+`setserial -b {{device}}`
+
+- Set configuration parameters to a device by passing commandline arguments. Superuser privilege is required in most cases:
+
+`sudo setserial {{device}} {{parameter}}`
+
+- Read configuration of a list of devices by using the `-g` option:
```
Comment (navarroaxel) [inline pages/linux/setserial.md:18]: The `-g` is specified in the example
```suggestion
- Print configuration of a list of devices:
```

File: pages/linux/setserial.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- Print all information of the supplied serial device (usually of the form /dev/cua[0-3]):
+
+`setserial -a {{device}}`
+
+- Print a summary of confguration of the serial device (useful for printing during bootup process):
+
+`setserial -b {{device}}`
+
+- Set configuration parameters to a device by passing commandline arguments. Superuser privilege is required in most cases:
```
Comment (navarroaxel) [inline pages/linux/setserial.md:14]: All the tldr examples are for the command-line
```suggestion
- Set a specific configuration parameter to a device:
```

File: pages/linux/setserial.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# setserial
+
+> Read or modify a LInux serial port information.
+> More information: <https://linux.die.net/man/8/setserial>.
+
+- Print all information of the supplied serial device (usually of the form /dev/cua[0-3]):
```
Comment (navarroaxel) [inline pages/linux/setserial.md:6]: We can applied the format to the example itself
```suggestion
- Print all information of a serial device:
```

File: pages/linux/setserial.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`setserial -a {{device}}`
```
Comment (navarroaxel) [inline pages/linux/setserial.md:8]:
```suggestion
`setserial -a {{/dev/cua0}}`
```

File: pages/linux/setserial.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+- Print a summary of confguration of the serial device (useful for printing during bootup process):
```
Comment (navarroaxel) [inline pages/linux/setserial.md:10]:
```suggestion
- Print a summary of configuration of a specific serial device (useful for printing during bootup process):
```

File: pages/linux/setserial.md (hunk 1 above)
Comment (DEBADRIBASAK) [reply pages/linux/setserial.md]: Done.

File: pages/linux/setserial.md (hunk 3 above)
Comment (DEBADRIBASAK) [reply pages/linux/setserial.md]: Done.

**Comment by DEBADRIBASAK** [comment]
@navarroaxel addressed the suggestions.

**Comment by navarroaxel** [comment]
> @navarroaxel addressed the suggestions.

@DEBADRIBASAK I think you didn't push the commit. :grimacing:

File: pages/linux/setserial.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
+# setserial
+
+> Read and modify serial ports information.
```
Comment (kbdharun) [inline pages/linux/setserial.md:3]:
```suggestion
> Read and modify serial port information.
```

**Comment by DEBADRIBASAK** [comment]
@navarroaxel I pushed the commits.

File: pages/linux/setserial.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
+# setserial
+
+> Read and modify serial port information.
+> More information: <https://manned.org/setserial>.
+
+- Print all information of a serial device:
+
+`setserial -a {{/dev/cua0}}`
```
Comment (navarroaxel) [inline pages/linux/setserial.md:8]:
```suggestion
`setserial -a {{/dev/cuaN}}`
```

**Review by navarroaxel — APPROVED** [review]

File: pages/linux/setserial.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
+# setserial
+
+> Read and modify serial port information.
+> More information: <https://manned.org/setserial>.
+
+- Print all information of a serial device:
```
Comment (CleanMachine1) [inline pages/linux/setserial.md:6]:
```suggestion
- Print all information about a specific serial device:
```

**Review by CleanMachine1 — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, after @CleanMachine1's suggestion.

**Pull request merged**